src/common/config/classes.ts

```diff
@@ -943,6 +979,19 @@ export class AppConfig {
       }
     }
 
+    // Use environment variables to fill homeassistant notification config if present
+    const { HOMEASSISTANT_TOKEN } = process.env;
+    if (HOMEASSISTANT_TOKEN) {
+      const homeassistant = new HomeassistantConfig();
+      homeassistant.token = HOMEASSISTANT_TOKEN;
```

Comment:
You'll need to declare, check in the `if` condition, and assign the `HOMEASSISTANT_LONG_LIVED_ACCESS_TOKEN`, `HOMEASSISTANT_INSTANCE`, and `HOMEASSISTANT_NOTIFYSERVICE` variables here.

Check how the other notifiers do it for each environment variable for an example.

Reply:
I overlooked these, please see the commit below.
The notifications did work during my testing though. Just wondering, is it able to use the config without these declarations?

Reply:
Correct. The environment variables are just a backup option for people that aren't using the `config.json`.